docs/misc/operation/aws-ami.md

```diff
@@ -29,7 +33,7 @@ We generate the latest AMI every day around 6 pm KST. Once generated, the AMIs l
 
 ## Launch new EC2 instance with AMI in Amazon console
 
-When launching a new EC2 instace in the AWS console, you should select an AMI. Search for `kaia-mainnet` or `kaia-kairos` in the AMI search bar.
+When launching a new EC2 instace in the AWS console, you should select an AMI. Search for `kaia-mainnet` in the AMI search bar.
```

Comment:
small typo : instace to instance.

Reply:
Done, thanks!

Reply:
@ayo-klaytn Please take a look and approve this PR if you don't find any other issues.